start.R

```diff
@@ -50,7 +56,8 @@ invisible(sapply(list.files("scripts/start", pattern = "\\.R$", full.names = TRU
 # define arguments that are accepted
 acceptedFlags <- c("0" = "--reset", "1" = "--testOneRegi", d = "--debug", g = "--gamscompile", i = "--interactive",
                    r = "--restart", R = "--reprepare", t = "--test", h = "--help", q = "--quick")
-flags <- lucode2::readArgs(.flags = acceptedFlags, .silent = TRUE)
+startgroup <- "1"
```

Comment:
I think it would be advisable to rename `startnow` to `startgroup` also in `start_bundle_coupled.R` to improve consistency.

Reply:
done, coupled tests are runnning.